ee/ucp/release-notes.md

```diff
@@ -36,7 +36,12 @@ upgrade your installation to the latest release.
 * Enabled Kubernetes Node Authorizer Plugin. (ENGORC-2652)
 
 ### Networking
-* Interlock has been upgraded to version 3.0.0. (ENGCORE-792)
+- Interlock has been upgraded to version 3.0.0. This upgrade includes the following updates:
+  - New Interlock configuration options:
+    - HitlessServiceUpdate: When set to `true`, the proxy service no longer needs to restart when services are updated, reducing service interruptions. The proxy also does not have to restart when services are added or removed, as long as the set of service networks attached to the proxy is unchanged. If secrets or service networks need to be added or removed, the proxy service will restart as in previous releases. (ENGCORE-792)
+    - Networks: Defines a list of networks to which the proxy service will connect at startup. The proxy service will only connect to these networks and will no longer automatically connect to back-end service networks. This allows administrators to control which networks are used to connect to the proxy service and to avoid unnecessary proxy restarts caused by network changes . (ENGCORE-912)
+  - Log an error if the `com.docker.lb.network` label does not match any of the networks to which the service is attached. (ENGCORE-837)
+  - Do not generate an invalid NGINX configuration file if HTTPVersion is invalid. (FIELD-2046)
```

Comment:
Is there a space missing between HTTP and version? Not sure if HTTPVersion is a parameter. If it is a parameter, should we format it?